tests/functional/test_reinstall_clean.py

```diff
@@ -105,7 +105,7 @@ async def fixture_install_flow(
     [
         (
             'opt/rose-suite-cylc-install.conf', (
-                '# This file records CLI Options.\n\n'
+                '# This file records CLI Options.{version_info}\n'
```

Comment:
This isn't a format string.

The test still passes, so the `version_info` is presumably not require here?